modules/oe_paragraphs_media_field_storage/oe_paragraphs_media_field_storage.install

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @file
+ * OE Paragraphs Media Field Storage install.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Implements hook_install().
+ */
+function oe_paragraphs_media_field_storage_install() {
+  // The storage of field_oe_media was initially used only in the "Text with
+  // Featured Media" paragraph but we created a new module that installs
+  // "Banner" paragraph where the field_oe_media storage is required as well.
+  // We moved the storage of the field within this module under the
+  // /config/import folder and import it with an install hook in order to avoid
+  // errors when enabling this module on sites where this configuration already
+  // exists.
+  $field_storage = Drupal::entityTypeManager()->getStorage('field_storage_config')->load('paragraph.field_oe_media');
+  if ($field_storage) {
+    return;
+  }
+
+  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs_media_field_storage') . '/config/import');
+  $config_id = 'field.storage.paragraph.field_oe_media';
+  $config_record = $storage->read($config_id);
+  $config_entity_type = \Drupal::service('config.manager')->getEntityTypeIdByName($config_id);
+  $entity_storage = \Drupal::entityTypeManager()->getStorage($config_entity_type);
```

Comment:
I am pretty sure this and $field_storage are the same, so  we don't need to regenerate it (double check first)